db-resources/MongoDB/index.js

```diff
@@ -0,0 +1,93 @@
+// Connecting to MongoDB without Mongoose ORM
+const { MongoClient } = require('mongodb'), format = require('util').format;
+const { getRandomArbitrary, getRandomWhole, scoreRound, getRandomPcnt, generateRandomCountry, getProductId } = require('./seedHelper')
+const { performance } = require('perf_hooks')
+
+MongoClient.connect('mongodb://127.0.0.1:27017', { useNewUrlParser: true }, (err, client) => {
+  if (err) {
+    console.log('error', err)
+    client.close();
+  } else {
+    console.log('Connected');
+    const db = client.db('newegg');
+    const product = db.collection('product')
```

Comment:
This is where you actually connect to the Database.  The `db = client.db('newegg');` is the actual connection. Client is the connection to the server.

Move your connected log below db

Reply:
Done!